wallet/db.c

```diff
@@ -1072,6 +1075,51 @@ static void migrate_pr2342_feerate_per_channel(struct lightningd *ld, struct db
 	tal_free(stmt);
 }
 
+/* We've added a column `our_funding_satoshis`, since channels can now
+ * have funding for either channel participant. We need to 'backfill' this
+ * data, however. We can do this using the fact that our_funding_satoshi
+ * is the same as the funding_satoshi for every channel where we are
+ * the `funder`
+ */
+static void migrate_our_funding(struct lightningd *ld, struct db *db)
+{
```

Comment:
Pretty sure this is easy to do in SQL?  Since you set the default to 0, we just need to fix up the others?

"UPDATE channels SET our_funding_satoshi = funding_satoshi WHERE funder = 0;" /* 0 == LOCAL */

Reply:
Nice. Wishing I had written a test for this though >.< lol